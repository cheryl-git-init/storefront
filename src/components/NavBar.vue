<template>
  <div id="nav">
    <router-link to="/">
      <img src="@/assets/sign.svg" height="80px" />
      <h1>Home</h1>
    </router-link>
    <NavButtonBasket />
  </div>
</template>

<script>
// @ is an alias to /src
import NavButtonBasket from "@/components/NavButtonBasket.vue";

export default {
  name: "NavBar",
  components: {
    NavButtonBasket
  }
};
</script>

<style lang="scss" scoped>
img {
  float: left;
  margin: 10px;
}
h1 {
  float: left;
}
/*  GO FULL WIDTH BELOW 480 PIXELS */
@media only screen and (max-width: 481px) {
  h1 {
    display: none;
  }
}
</style>
