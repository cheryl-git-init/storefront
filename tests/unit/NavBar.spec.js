import NavBar from '@/components/NavBar.vue';
import { shallowMount } from '@vue/test-utils';


describe('NavBar.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(NavBar, {
        stubs: ['router-link']
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
