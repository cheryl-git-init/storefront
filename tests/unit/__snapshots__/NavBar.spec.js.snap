// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar.vue matches snapshot 1`] = `
<div
  id="nav"
>
  <router-link-stub
    to="/"
  >
    <img
      height="50px"
      src="@/assets/sign.svg"
    />
     
    <h1>
      Home
    </h1>
  </router-link-stub>
   
  <navbuttonbasket-stub />
</div>
`;
